src/App/Init.php

```diff
@@ -47,32 +47,66 @@
 	return $memcached;
 }));
 
-// Set up login rate limiter
-$di->set('ratelimiter.login.flap', $di->lazyNew('BehEh\Flaps\Flap'));
+// Set up register rate limiter
+$di->set('ratelimiter.register.flap', $di->lazyNew(
+	'BehEh\Flaps\Flap',
+	array(
```

Comment:
Can we always use `[]` style arrays? not `array()`?
I need to update linting rules to enforce that